students/olszewski_bartosz/lesson_06_dictionaries/uppercase.py

```diff
@@ -0,0 +1,16 @@
+a = input()
+
+
+def capitalize(sentence):
+    for i in range(len(sentence)):
+        if i == 0:
+            b = ord(sentence[i]) - 32
```

Comment:
You can use dedicated function for it
title = lower_case_word.title()
or
title = lower_case_word[0].upper() + lower_case_word[1:]